xsgen/openmc_origen.py

```diff
@@ -151,15 +152,16 @@ def __init__(self, rc):
         self.eafds = data_source.EAFDataSource()
         self.omcds = data_source.OpenMCDataSource(
                         cross_sections=rc.openmc_cross_sections,
-                        src_group_struct=np.logspace(1, -9, 1001))
+                        src_group_struct=rc.openmc_group_struct)
         data_sources = [self.omcds]
-        if not rc.is_thermal:
-            data_sources.append(self.eafds)
+        #if not rc.is_thermal:
+        #    data_sources.append(self.eafds)
```

Comment:
This needs to be resolved one way or the other before merging. I am leaning slightly towards removing it all together, since EAF is really for fussion reactors.